src/main/java/com/example/demo/database/Receipt.java

```diff
@@ -0,0 +1,47 @@
+package com.example.demo.database;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+import java.util.List;
+
+@Setter
+@Getter
+@Entity
+@Table(name = "receipts")
+public class Receipt {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long receiptId;
+
+    private long customerId;
+
+    private int totalAmount;
+
+    private Date dateTime;
```

Comment:
1. use `java.time.LocalDate` instead of `java.util.Date`
2. think of a better name for the variable. What does this `dateTime` mean?